roles/core/nfs_client/molecule/default/molecule.yml

```diff
@@ -0,0 +1,26 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance
+    groups:
+      - mg_computes
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+provisioner:
+  name: ansible
+  ansible_args:
+    - -e mount_exported_dirs=false
```

Comment:
We are used to define global vars in converge.yml ([example](https://github.com/bluebanquise/bluebanquise/blob/master/roles/core/log_client/molecule/default/converge.yml)).

Reply:
Yes, I used to declare variables in converge.yml too, however here it is not working.
I have declared directly the variable like an extra var to have the highest priority.
I will double chek that.

Reply:
Corrected in commit f28b555